src/daria/image/image.py

```diff
@@ -175,6 +192,8 @@ def write(
             metadata_path (str): path to metadata (only folders); the metadata file
                 has the same name as the image (and a .txt ending)
         """
+        # TODO need to make sure that image is in BGR format
```

Comment:
Can't you just make a conditional statement to self.colorspace?

Reply:
Are you finished with defining a flag for colorspaces? I did not notice that. Therefore, I was just adding a todo note.